packages/renderer/src/lib/container/ContainerList.svelte

```diff
@@ -313,11 +319,17 @@ let nameColumn = new TableColumn<ContainerInfoUI | ContainerGroupInfoUI>('Name',
 
 let envColumn = new TableColumn<ContainerInfoUI | ContainerGroupInfoUI>('Environment', {
   renderer: ContainerColumnEnvironment,
+  comparator: (a, b) => (a.engineType ?? '').localeCompare(b.engineType ?? ''),
 });
 
 let imageColumn = new TableColumn<ContainerInfoUI | ContainerGroupInfoUI>('Image', {
   width: '3fr',
   renderer: ContainerColumnImage,
+  comparator: (a, b) => {
+    const aImage = 'image' in a ? a.image : '';
```

Comment:
Same remark

Reply:
@jeffmaury same error as in the previous comment